### ISSUE TITLE
QuPath version parsing

### ISSUE THREAD

**Issue by pwalczysko** [glencoesoftware/ome-omero-roitool#27]
When the ``OME_XML_export.groovy`` v0.2.1 is run in QuPath v0.2.1, then the exception below occurs.
Is that because of the parsing of the version in https://github.com/glencoesoftware/ome-omero-roitool/blob/master/src/dist/QuPath.scripts/OME_XML_export.groovy#L60 ?

Note: the same script works fine in QuPath v0.2.0

cc @melissalinkert @jburel @chris-allan 

```
ERROR: RuntimeException at line 61: Unsupported QuPath version: 0.2.1

ERROR: java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
    java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
    java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
    java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
    java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
    org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)
    org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)
    org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
    org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
    org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
    Script1.run(Script1.groovy:62)
    org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.eval(GroovyScriptEngineImpl.java:317)
    org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.eval(GroovyScriptEngineImpl.java:155)
    qupath.lib.gui.scripting.DefaultScriptEditor.executeScript(DefaultScriptEditor.java:926)
    qupath.lib.gui.scripting.DefaultScriptEditor.executeScript(DefaultScriptEditor.java:859)
    qupath.lib.gui.scripting.DefaultScriptEditor.executeScript(DefaultScriptEditor.java:782)
    qupath.lib.gui.scripting.DefaultScriptEditor$2.run(DefaultScriptEditor.java:1271)
    java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
    java.base/java.util.concurrent.FutureTask.run(Unknown Source)
    java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
    java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
    java.base/java.lang.Thread.run(Unknown Source)
```

**Comment by melissalinkert** [comment]
https://github.com/glencoesoftware/ome-omero-roitool/pull/28 should fix this.

**Comment by chris-allan** [comment]
Fixed by #28. Released in 0.2.2.
